Pellicola/Classes/ImagesDataStorage.swift

```diff
@@ -23,10 +22,12 @@ class OrderedImage: NSObject, Comparable {
 
 class ImagesDataStorage: NSObject {
     
-    @objc dynamic private(set) var images: [String: OrderedImage] = [String: OrderedImage]()
+    @objc dynamic private(set) var images = [String: OrderedImage]()
     private var index = 0
     
     let limit: Int?
+    let fileQueue = DispatchQueue(label: "com.subito.pellicola.files")
```

Comment:
What about moving this queue within FileHandler itself? This way clients of its API would not need to worry about synching.